optuna/integration/cma.py

```diff
@@ -30,16 +32,18 @@
     from optuna.trial import FrozenTrial  # NOQA
     from optuna.study import Study  # NOQA
 
+_logger = logging.get_logger(__name__)
+
 # Minimum value of sigma0 to avoid ZeroDivisionError in cma.CMAEvolutionStrategy.
 _MIN_SIGMA0 = 1e-10
 
 
-class CmaEsSampler(BaseSampler):
+class PyCmaSampler(BaseSampler):
```

Comment:
Not a strong opinion but naming it `CmaSampler` seems more consistent with `SkoptSampler`, based on the module names at import time. However, it might be controversial as I believe the name `SkoptSampler` has been brought to discussion before.

Reply:
I think `PyCmaSampler` matches with the library name `pycma` as seen in https://github.com/CMA-ES/pycma. It is easy to understand what the CMA-ES library used in this sampler. 

I also feel the `CmaSampler` and `CmaEsSampler` are too similar. I'm afraid to mistake them.